app/controllers/api/v1/vocabularies_controller.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+###
+# @description: Place all the actions related to vocabularies
+###
+class Api::V1::VocabulariesController < ApplicationController
+  before_action :authorize_with_policy
+
+  include Pundit
+
+  ###
+  # @description: Returns the list of vocabularies for the logged in
+  #   user's organization
+  ###
+  def index
+    vocabularies = Vocabulary.all.order(name: :desc)
+    render json: vocabularies
+  end
+
+  ###
+  # @description: Creates a vocabulary
+  ###
+  def create
+    @vocabulary = Vocabulary.create!(valid_params)
+
+    render json: {
+      success: true,
+      vocabulary: @vocabulary
+    }
+  end
+
+  private
+
+  ###
+  # @description: Execute the authorization policy
+  ###
+  def authorize_with_policy
+    authorize vocabulary
+  end
+
+  ###
+  # @description: Get the current model record
+  # @return [ActiveRecord]
+  ###
+  def vocabulary
```

Comment:
This action doesn't seem to have a route.

Reply:
@excelsior It's not meant to have it. 

As mentioned above, this method, alongside with `authorize_with_policy` works to let the Pundit gem authorize the model. I do it like this to avoid having it repeated in every controller method. It's also marked as private, so it's not accessible from outside the class.

If you think this needs to be in a separate class, I'd have to rethink how to do it without repeating the authorize line in every method of this (and the other) controllers.

Reply:
My bad, didn't notice that.